src/app/_components/page-assets/countdown.tsx

```diff
@@ -0,0 +1,101 @@
+"use client";
+import Section from "./section";
+import { Skeleton } from "../ui/skeleton";
+import { useEffect, useState } from "react";
+
+export default function Countdown() {
+  //remove usage of trpc; reliant on local machine time per render
```

Comment:
The idea was to call the tRPC route: `getTimeRemaining` in order to get either the date of the event / time to countdown to or the difference of time between now and the future time.

1. Fire off the tRPC request and set this as the initial state for `time`.
2. Once the initial state is received, then start the "countdown" process on the client

Or, you can make this a purely client-side component, as you've done here.